components/up-job.libsonnet

```diff
@@ -0,0 +1,79 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+{
+  local up = self,
+
+  config:: {
+    name: error 'must provide name',
+    version: error 'must provide version',
+    image: error 'must provide image',
+    backoffLimit: error 'must provide backoffLimit',
+    writeEndpoint: error 'must provide writeEndpoint',
+    readEndpoint: error 'must provide readEndpoint',
+
+    commonLabels:: {
+      'app.kubernetes.io/name': 'observatorium-up',
+      'app.kubernetes.io/instance': up.config.name,
+      'app.kubernetes.io/version': up.config.version,
+      'app.kubernetes.io/component': 'test',
+    },
+  },
+
+  job:
+    local job = k.batch.v1.job;
+    local container = job.mixin.spec.template.spec.containersType;
+
+    local c =
+      container.new('observatorium-up', up.config.image) +
+      container.withArgs([
+        '--endpoint-write=' + up.config.writeEndpoint,
```

Comment:
How to configure how often those tests are performed? :thinking:

Reply:
it's defined by `duration=2m` meaning we will probe for a total of 2 minutes in 1s periods and 90% write must have a read-after-write latency of less than 10s

Reply:
Ok, but how often the test is running? Constantly?

Reply:
every 1 second for 2 minutes, so ~120 times